applications/poisson/sine/application.h

```diff
@@ -123,6 +123,9 @@ class Application : public ApplicationBase<dim, n_components, Number>
     prm.enter_subsection("Application");
     {
       prm.add_parameter("MeshType", mesh_type, "Type of mesh (Cartesian versus curvilinear).");
+      prm.add_parameter("Preconditioner",
+                        preconditioner,
+                        "Preconditioner employed for implicit solution procedure.");
```

Comment:
What do you mean by `implicit solution procedure` in the context of Poisson?

Reply:
As we don't have a temporal discretization here, I would suggest something like `Preconditioner for the linear system`. I think explicit/implicit is confusing in this context.